extensions/google-translate/src/actions.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { Action, getPreferenceValues } from "@raycast/api";
+
+enum DefaultActionPreference {
+  CopyToClipboard = "copy",
+  PasteToApp = "paste",
+}
+
+interface Preferences {
+  defaultAction?: DefaultActionPreference;
+}
+
+interface ActionsOpts {
+  value: string;
+  defaultActionsPrefix?: string;
+  firstSectionTitle?: string;
+  otherActions?: React.ReactElement[];
+  otherSections?: React.ReactElement[];
```

Comment:
ah and let's remove this 3 params, as it is not used now

and rename `ActionsOpts` to `ConfigurableCopyPasteActionsProps`